.yarn/patches/rn-fetch-blob-npm-0.12.0-cf02e3c544.patch

```diff
@@ -0,0 +1,115 @@
+diff --git a/android/src/main/java/com/RNFetchBlob/RNFetchBlobConfig.java b/android/src/main/java/com/RNFetchBlob/RNFetchBlobConfig.java
```

Comment:
Thanks for fixing this! Could you please add a comment at the top explaining what the patch is for? I think you can add comments by prefixing them with `#`